releases/adm/sciencebeam-alert-rules.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: sciencebeam.rules
+  namespace: adm
+  labels:
+    app: sciencebeam
+spec:
+  groups:
+  - name: sciencebeam
```

Comment:
most existing rules do not seem to be mentioning rules in group name. Happy to change to whatever seems best.

Reply:
LGTM. I think I misunderstood the convention when I was adding rules.